spec/features/create_short_url_spec.rb

```diff
@@ -2,13 +2,56 @@
 
 describe "creating a short url" do
   context "when not signed in" do
-  end
+    before do
+      visit "/home"
+      fill_in "link[long_url]", :with => "http://www.msn.com"
+      click_button "Shorten It"
+    end
+    
+    it "should create a record for the new URL in the database" do
+      expect(Link.last.long_url).to eq("http://www.msn.com")
+      expect(Link.last.short_url).not_to be_nil 
```

Comment:
For lines 12-13:

Testing if an object is created and has certain attributes (something that happens in the model and controller) should be tested in functional tests.  The user doesn't care that they are in the database :smile: